fdbserver/workloads/Watches.actor.cpp

```diff
@@ -177,87 +169,76 @@ struct WatchesWorkload : TestWorkload {
 		}
 	}
 
-	ACTOR static Future<Void> watchesWorker(Database cx, WatchesWorkload* self) {
-		state Key startKey = self->keyForIndex(self->nodeOrder[0]);
-		state Key endKey = self->keyForIndex(self->nodeOrder[self->nodes]);
-		state Optional<Value> expectedValue;
-		state Optional<Value> startValue;
-		state double startTime = now();
-		state double chainStartTime;
+	Future<Void> watchesWorker(Database cx, WatchesWorkload* self) {
+		Key startKey = self->keyForIndex(self->nodeOrder[0]);
+		Key endKey = self->keyForIndex(self->nodeOrder[self->nodes]);
+		Optional<Value> expectedValue;
+		Optional<Value> startValue;
+		double startTime = now();
+		double chainStartTime;
 		loop {
```

Comment:
Can this `loop` also be translated to native C++ code?

Reply:
yes. it's equivalent to `while (true)`. maybe in another PR.